frontend/src/scenes/insights/Trends.cy-spec.js

```diff
@@ -0,0 +1,353 @@
+            cy.contains('http://posthog.com').should('be.visible')
+        })
+
+        it('responsive to shown as', () => {
+            helpers.setLocation('/insights', {
+                insight: 'TRENDS',
+                interval: 'day',
+                display: 'ActionsLineGraph',
+                events: [
+                    {
+                        id: '$pageview',
+                        name: '$pageview',
+                        type: 'events',
+                        order: 0,
+                    },
+                ],
+                properties: [],
+                shown_as: 'Stickiness',
+            })
+            mount()
+            cy.wait('@api_insight')
+                .map(helpers.getSearchParameters)
+                .should('include', {
+                    insight: 'TRENDS',
+                    interval: 'day',
+                    display: 'ActionsLineGraph',
+                    events: JSON.stringify([
+                        {
+                            id: '$pageview',
+                            name: '$pageview',
+                            type: 'events',
+                            order: 0,
+                        },
+                    ]),
+                    properties: '[]',
+                    shown_as: 'Stickiness',
+                })
+            cy.get('[data-attr="trend-line-graph"]').should('be.visible')
+            cy.contains('Stickiness').should('be.visible')
+        })
+
+        it('responsive to breakdown', () => {
+            helpers.setLocation('/insights', {
+                insight: 'TRENDS',
+                interval: 'day',
+                display: 'ActionsLineGraph',
+                events: [
+                    {
+                        id: '$pageview',
+                        name: '$pageview',
+                        type: 'events',
+                        order: 0,
+                    },
+                ],
+                properties: [],
+                breakdown: '$browser',
+                breakdown_type: 'event',
+            })
+            mount()
+
+            cy.wait('@api_insight')
+                .map(helpers.getSearchParameters)
+                .should('include', {
+                    insight: 'TRENDS',
+                    interval: 'day',
+                    display: 'ActionsLineGraph',
+                    events: JSON.stringify([
+                        {
+                            id: '$pageview',
+                            name: '$pageview',
+                            type: 'events',
+                            order: 0,
+                        },
+                    ]),
+                    properties: '[]',
+                    breakdown: '$browser',
+                    breakdown_type: 'event',
+                })
+            cy.get('[data-attr="trend-line-graph"]').should('be.visible')
+            cy.contains('$browser').should('be.visible')
```

Comment:
Hmm. Reading the test not sure what this line is intending to test for - at the moment it's looking for the word $browser anywhere. Can we narrow it down?